### PR TITLE
treewide: srgom from meta.maintainers

### PR DESCRIPTION
**Pull request by SuperSandro2000**
The github account has been inactive for 4+ years so I have no hope that the person is coming back.

## Description of changes

I also don't agree with the closing of #337369. The account has been inactive for over 4 years and at this point we are just lying about the maintenance status of the packages.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by JohnRTitor** [comment]
> at this point we are just lying about the maintenance status of the packages

Exactly.

**Review by JohnRTitor — APPROVED** [review]

**Comment by bendlas** [comment]
> I also don't agree with the closing of #337369.

Figures. This PR has almost all of the same problems as that one:
- no link to any kind of overarching consensus or at least discussion of how maintainer inactivity is to be handled and how that would justify this removal
- no thank you or other personal note to the maintainer being removed (even as a standardized text block)
- no advice on how to pick maintainership back up
- just an off the cuff remark from supersandro2000, and one that could be read as disparaging at that
- otherwise just the standard PR template, geared towards package builds

I don't want this to be the way we treat our maintainers. Even long-time inactive ones. Even if they might never read it.

**Review by superherointj — APPROVED** [review]

**Comment by Artturin** [comment]
Squash merged with

treewide: Retire `srgom` from meta.maintainers

The GitHub account has been inactive for 4+ years.

This is sadly necessary due to the delays in reviews/merges when others are waiting for the requested maintainers to review the PR.

Instructions for re-adding once you return https://github.com/NixOS/nixpkgs/tree/master/maintainers#how-to-become-a-maintainer

We appreciate your past and future contributions.

**Pull request merged**
